AirSim/docs/ros.md

```diff
@@ -0,0 +1,58 @@
+# How to use AirSim with Robot Operating System (ROS)
+
+AirSim and ROS can be integrated using C++ or Python.  Some example ROS nodes are provided demonstrating how to publish data from AirSim as ROS topics.
+
+# Python
+
+## Prerequisites
+
+These instructions are for Ubuntu 16.04, ROS Kinetic, UE4 4.18 or higher, and latest AirSim release.
+You should have these components installed and working before proceeding
```

Comment:
Did we write this doc? From the looks of it we need to consider removing this from the documentation or updating it (we are working on ubuntu 18.04 and ros melodic)

Reply:
nope, I accidently comitted some junk when trying to figure out how to get the changes for our workspace, there will be a cleanup commit

Reply:
Addressed 0e7b80f